### DISCUSSION TITLE
[Bug] Notifications system false alert on non-exist messages

### DISCUSSION THREAD

**Discussion by CNLHC** [community/community#111283] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Github has a `Notification` button placed at the top right of the homepage. Github will use a small blue point to indicate me that i have some unread notifications. 
<img width="284" alt="image" src="https://github.com/community/community/assets/21005146/3fcf4d2c-d001-436a-a642-ae9d4a78adec">

But if i enter the `notifications`, weird things happen. The page shows that i have one unread message, which is also categorized as `participating` and `mentioned`. But the right part of page show no message at all.  I tried to tweak the filter settings but still can not see any messages.  

<img width="1018" alt="image" src="https://github.com/community/community/assets/21005146/51a0775f-cd7d-4793-a70c-d205c4316175">

Now my github page will always suggest that I have an unread message that does not exist and can not be read.

**Comment by himeshx** [comment]
Hey there are a few troubleshooting steps you can try if you're facing a false alert bug -
-> Clear Browser Cache : Sometimes, the issue can be due to cached data. Try clearing your browser cache and then reloading the GitHub page to see if the problem persists;
-> Different Browser or Incognito Mode : Try accessing GitHub using a different browser or using incognito mode in your current browser. This can help determine if the issue is specific to your current browser or a more general problem.
-> Use GitHub API to Check Notifications : If you are familiar with using the GitHub API, you can use it to check your notifications. This can sometimes provide more details than the web interface.
You can use the following endpoint:

GET /notifications

This will return a list of your notifications. You might be able to see the unread notification that is causing the issue.

If none of the above steps work, it might be a bug or a more specific issue related to your account. Try reaching out to GitHub You can do this by clicking on "Contact GitHub" at the bottom of the page or by visiting GitHub Support.

ps: Kindly consider marking the provided response as the answer if it has addressed your query satisfactorily.

**Reply by CNLHC** [reply]
Thanks for your suggestions.  I have tried your suggestions and

1. Clear Browser Cache: not work
2. Different Browser Or Incognito Mode: not work
3. Try the github API

```bash
curl -L \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/notifications\?all\=false 
```

Return empty list, and if i change `all` to true, i can see all my readed notifications.

**Comment by FR98** [comment, marked as answer]
Hello, I got this same error after being notified about Starknet and Fluence coins airdrop by email. Looks like is related to this: https://github.com/orgs/community/discussions/6874#discussioncomment-8687949

I think that we all were mentioned on a gist, but I wasn't able to look the content.

I already contact Github support and they told me the same about using curl to clear the notifications manually with the API.

**Reply by CNLHC** [reply]
I think you are right.

For other people who read this answer, please use API to clear your notification box.

```bash
curl -X PUT \
    -H "Accept: application/vnd.github.v3+json" \
    -H "Authorization: token <TOKEN>" \
     https://api.github.com/notifications \
    -d '{"last_read_at":"2024-03-08T00:00:00Z"}'
```

**Reply by Aviksaikat** [reply]
what token should be here?

**Reply by CNLHC** [reply]
Your github personal access token, generate from your settings page.

**Reply by Aviksaikat** [reply]
Awesome thanks

**Comment by Vonfry** [comment]
I meet this issue recently as well, and temporarily fix it by api from CNLHC's method. IMO, github should check the codebase to fix source.

**Comment by Aviksaikat** [comment]
Facing this issue as well
